packages/payment-integration-api/src/index.ts

```diff
@@ -26,7 +26,13 @@ export { Omit, PartialDeep } from './common/types';
 export { objectWithKebabCaseKeys, AmountTransformer } from './common/utility';
 export { Config, StoreConfig, CheckoutSettings, StoreProfile } from './config';
 export { Coupon } from './coupon';
-export { Currency, CurrencyFormatter, CurrencyConfig } from './currency';
+export {
+    Currency,
+    CurrencyFormatter,
+    CurrencyConfig,
+    CurrencyService,
+    createCurrencyService,
```

Comment:
I am unsure about exposing this. As this is an internal for checkout-sdk and in theory should not be referenced in packages.

Reply:
I am not sure that it is only internal for checkout-sdk, because we are using it in checkout-js as well.
And I need it for showing relevant price for shipping options inside GPay modal window.

Reply:
In an ideal state we can pass an instance of it, rather than creating another instace for google pay but seems ok.